tests/acceptance/build.sh

```diff
@@ -26,6 +26,41 @@ if [ $(echo ${#fmtResult}) != 0 ]
 	else echo -e "${GREEN}√ go fmt${NC}"
 fi
 
+goFilesToCheck=$(find . -type f -name "*.go" | egrep -v "\/vendor\/")
+
+##
+# GO IMPORTS & FMT
+##
+go build -o goimports-vendored ./vendor/golang.org/x/tools/cmd/goimports
+buildGoImportResult=$?
+if [ ${buildGoImportResult} != 0 ]; then
+	echo -e "${RED}✗ go build goimports${NC}\n$buildGoImportResult${NC}"
+	exit 1
+fi
+
+goImportsResult=$(echo "${goFilesToCheck}" | xargs -L1 ./goimports-vendored -w -l)
+rm goimports-vendored
+
+if [ $(echo ${#goImportsResult}) != 0 ]; then
+	echo -e "${RED}✗ goimports and fmt${NC}\n$goImportsResult${NC}"
+	exit 1
+else echo -e "${GREEN}√ goimports and fmt${NC}"
+fi
+
+##
+# GO VET
+##
+packagesToVet=("./dex/..." "./servicecatalog/..." "./remote-environment/...")
```

Comment:
if we add more tests, we also need to change that. Please check if it is another option to do that.